### PR TITLE
Cluster Overloaded

### PR DESCRIPTION
**Pull request by tnierman**
Adds a new message for cases where cluster CPU is insufficient.

### CODE REVIEW

File: osd/cpu_overloaded.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+ "severity": "Warning",
+ "service_name": "SREManualAction",
+ "cluster_uuid": "${CLUSTER_UUID}",
+ "summary": "Action required: CPU overloaded",
+ "description": "Your OpenShift Dedicated cluster does not have enough CPU resources and requires you to take action. You need to either reduce application load or increase worker nodes. See the following documentation for scaling up worker nodes: https://docs.openshift.com/dedicated/4/getting_started/scaling-your-cluster.html",
```
Comment (dustman9000) [inline osd/cpu_overloaded.json:6]: Can we make this a little more generic (CPU + Memory)? I think just removing "CPU" will do it.

Comment (tnierman) [reply osd/cpu_overloaded.json]: Good idea

File: osd/cluster_overloaded.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+ "severity": "Warning",
+ "service_name": "SREManualAction",
+ "cluster_uuid": "${CLUSTER_UUID}",
+ "summary": "Action required: CPU overloaded",
```
Comment (dustman9000) [inline osd/cluster_overloaded.json:5]:
```suggestion
 "summary": "Action required: Resources overloaded",
```

**Comment by dustman9000** [comment]
Thanks for adding this!
/lgtm

**Pull request merged**